src/index.tsx

```diff
@@ -16,6 +17,10 @@ if (process.env.NODE_ENV === 'production') {
   log.setLevel(log.levels.DEBUG, false);
 }
 
+if (process.env.REACT_APP_SENTRY_DSN) {
+  Raven.config(process.env.REACT_APP_SENTRY_DSN).install();
```

Comment:
I still think having `logger` defined everywhere (client and server) makes the filtering easier but if you want to add that later, it's fine.